instaloader/__main__.py

```diff
@@ -14,6 +14,7 @@
 from .instaloader import (get_default_session_filename, get_default_stamps_filename)
 from .instaloadercontext import default_user_agent
 from .lateststamps import LatestStamps
+import browser_cookie3
```

Comment:
Since this is an optional dependency, it should be wrapped into a `try` block, that disables the new feature if that import is missing, e.g. by setting a variable in that case and then `--load-cookies` showing a message that `browser_cookie3` needs to be installed.

Reply:
I have made a commit that addresses this.